### DISCUSSION TITLE
how to enable nginx monitoring

### DISCUSSION THREAD

**Discussion by alisacorporation** [netdata/netdata#15130] [Q&A]
Hi all, can't figure out how to enable nginx monitoring.. Should i enable also plugin go.d or python.d? Should i configure log files etc... Can please someone explain for newbie, because i can't find any good explain doc/video.. Thanks!

**Comment by ilyam8** [comment]
Hi, @iBrabus. Have you configured [ngx_http_stub_status_module](http://nginx.org/en/docs/http/ngx_http_stub_status_module.html)? You just need to add `/basic_status` location, see the example in the ngx_http_stub_status_module documentation.

**Reply by alisacorporation** [reply]
yes i have configured, also putted in url in python.d/nginx.conf and tried to put it into go.d/nginx.conf

i need to know, should i need to manually enable python.d / go.d
should i need to provide paths for access_log / error_log etc.. there is no good exaplained instructions about it :(

**Reply by alisacorporation** [reply]
and i'm thinking, is nginx charts/monitorins available for netdata without cloud / signed in ? because im not using it

**Reply by alisacorporation** [reply]
ouchh, now entered after night in control and saw nginx in control, seems something started work..

ok i will start to look for more configuration about web logs etc.. thank you!

**Reply by andrewm4894** [reply]
maybe have a look at docs here too: https://www.netdata.cloud/nginx-monitoring/
